src/mono/mono/metadata/CMakeLists.txt

```diff
@@ -42,6 +42,11 @@ else()
 set(metadata_platform_sources ${metadata_unix_sources})
 endif()
 
+set(imported_native_sources
```

Comment:
> Blocked on getting SIMD enabled for mono/metadata on WASM. 

@kg  I /think/ just using `set_source_file_properties` to set compile flags would work:

```cmake
set_source_file_properites(${imported_native_sources} PROPERTIES COMPILE_FLAGS -msimd128)
```
under the appropriate `if()/endif()`.

It's possible this won't work if we're doing something weird with paths.

Reply:
The thing is we need to conditionally build with/without simd based on the msbuild property. Right now that's handled by building simd and non simd modules that get linked in dynamically at the end.

Reply:
do you think we could move the hash containers and stuff into their own object that gets linked late? i tried putting these things in our existing simd/no-simd objects, but i got link errors earlier in the build because it seems like we have a linking tree, i.e.

ab = a + b
cd = c + d
mono-sgen = ab + cd

Reply:
I think it should be possible to build the hash containers in their own object library (modulo anything inlined in headers).  I suspect it wouldn't work if you just do it in src/native/containers but src/mono/mono could probably do something.